docs/TypeScript.md

```diff
@@ -143,8 +143,8 @@ const opts: fastify.RouteShorthandOptions = {
 server.get<fastify.DefaultQuery, Params, unknown>('/ping/:bar', opts, (request, reply) => {
   console.log(request.query) // this is of type fastify.DefaultQuery!
   console.log(request.params) // this is of type Params!
-  console.log(request.body) // this is of type unknown!
-  console.log(request.headers) // this is of type fastify.DefaultHeader because typescript will use the default type value!
+  console.log(request.headers) // this is of type unknown!
+  console.log(request.body) // this is of type fastify.DefaultBody because typescript will use the default type value!
```

Comment:
This was the easiest correction to make to be valid, however I think it does take the value of example down a bit.
I thought the example was trying to make the case for `Body` being `unknown` and because no value was given for headers it defaults to `DefaultHeaders`; however, when the type ordering is corrected if we keep the example there is not a way to specify `unknown` for body without defining type for headers which means the example is no longer applicable. This is why I only switched the order.